infra-lifecycle-wg/charter/charter.md

```diff
@@ -0,0 +1,72 @@
+
+## Deliverable
+
+Develop a framework for managing the lifecycle of infrastructure in public, private, or hybrid cloud environments based on cloud-native principles. This framework will guide end-users in managing infrastructure to ensure it is secure, resilient, manageable, sustainable, and observable.
+
+* **Start by**, collaborating with relevant Technical Advisory Groups (TAGs), Working Groups (WGs), vendors and end-users to integrate domain-specific expertise.
+* **Next**, develop practical examples and reference architectures to guide end-users in applying the Infrastructure Lifecycle framework effectively across varied infrastructure environments.
+* **Finally**, publish real-world case studies and collect feedback from end-users on their experiences with applying the Infrastructure Lifecycle framework.
+
+The key categories for the framework are:
+* Infrastructure as Code
+  * Development processes
+  * Design and abstractions
+* Control planes
+* State management
+* Disaster recovery
+* Automation
+* Testing
+* Observability
+
+## Long-term vision
+After delivering the framework, we will assess its effectiveness and decide whether a longer-term Working Group or TAG is needed. This approach ensures the framework remains relevant and adaptable to evolving infrastructure management needs.
+
+## Out of Scope
+
+* This working group is not intended to endorse or promote any specific infrastructure management tools or technologies.
+* This working group won’t develop a new type of standard to infrastructure lifecycle management.
+* This working group won’t create a new CNCF open source project.
+
+## Examples of known patterns aimed to manage lifecycle of infrastructure
+
+### Infrastructure as Code Tools
+
+There are various tools enabling practitioners to define and manage the lifecycle of infrastructure through code, enabling consistent and scalable deployments of their cloud infrastructure. Tools like Terraform, CloudFormation, Pulumi and Crossplane allow practitioners to describe their infrastructure in a declarative manner, ensuring reliability in managing the lifecycle of cloud infrastructure.
+
+### Infrastructure Tool Orchestration
+
+The growing need for GitOps-style deployments drives demand for orchestrating declarative cloud infrastructure using Infrastructure as Code Tools. These orchestrators manage cloud resources either using native APIs or IaC tools. They offer flexibility across cloud providers and tooling preferences. Tools like TACoS, Crossplane, and AWS Controller for Kubernetes (ACK) exemplify this powerful approach, simplifying cloud infrastructure management.
+
+## Examples of known tools aimed to manage the lifecycle of infrastructure:
+
+Without any claim to comprehensiveness:
+
+* Ansible
+* AWS CloudFormation & AWS CDK
```

Comment:
Is this AWS cloud only?

Reply:
It's not an exhaustive list. We state on line 51: `Without any claim to comprehensiveness:`